### PR TITLE
Create profile

### PR DESCRIPTION
**Pull request by lakuleavadhutglobant**
Create profile Static Data binding done. Publish listing and Edit profile yet to be completed.

### CODE REVIEW

File: Application/Client/src/app/_common/header/components/header.component.ts [hunk 1]
```diff
@@ -20,10 +23,13 @@ export class HeaderComponent implements OnInit{
...
 
   constructor(public _authenticationWindowService: AuthenticationWindowService,
-              private _cookieService:CookieService){}
+              private _cookieService:CookieService,
+              private _route:Router,
+              public _base64Service:Base64Service ){}
 
   ngOnInit(){
     this.session = new Session( this._cookieService.getObject( 'SESSION_PORTAL' ) );
+    console.log("session email",this.session);
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/_common/header/components/header.component.ts]: remove this log

File: Application/Client/src/app/_common/header/components/header.component.ts [hunk 2]
```diff
@@ -38,6 +44,12 @@ export class HeaderComponent implements OnInit{
         console.log(event);
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/_common/header/components/header.component.ts]: stop priting larger logs

File: Application/Client/src/app/_common/similarListing/components/similarListing.component.ts [hunk 3]
```diff
@@ -15,33 +15,28 @@ export class SimilarListingComponent implements OnChanges{
...
 
   constructor(private httpService:CService,private apiPath:apiPaths){
-    
+
   }
 
   ngOnChanges(changes: {[propKey: string]: SimpleChange}){
-    console.log("changes");
-    console.log(changes); 
     this.getSimilarListing();
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/_common/similarListing/components/similarListing.component.ts]: Why do we need to call every change similer listing function?

File: Application/Client/src/app/_common/similarListing/components/similarListing.component.ts [hunk 4]
```diff
@@ -15,33 +15,28 @@ export class SimilarListingComponent implements OnChanges{
...
   }
 
   getSimilarListing(){
-    console.log("type is",this.type)
-    //let filter = '';
     let pathUrl= '';
     let filter = this.type.split("-");
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/_common/similarListing/components/similarListing.component.ts]: remove this variables are are not used anywhere also scope of this variables are only in function

File: Application/Client/src/app/card-list/components/card-list.component.ts [hunk 5]
```diff
@@ -21,17 +21,17 @@ export class CardListComponent{
     @Input() cards;
 
     private isLoading:boolean = false;
-    
+
     ngOnInit() {
         console.log('all = ',this.cards);
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/card-list/components/card-list.component.ts]: Remove this

File: Application/Client/src/app/createCard/component/createCard.component.ts [hunk 6]
```diff
@@ -191,22 +175,20 @@ export class CreateCardComponent implements OnInit {
            console.log("Finally");
          })
         this.submitted = true;
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/createCard/component/createCard.component.ts]: This flag set should be in success or failure calls

File: Application/Client/src/app/createProfile/component/createProfile.component.ts [hunk 7]
```diff
@@ -1,10 +1,11 @@
...
-import { FormGroup, FormControl, FormBuilder, Validators } from '@angular/forms';
 import {CService} from  '../../_common/services/http.service';
+import {SettingsService} from '../../_common/services/setting.service';
 import {mapData} from  '../../mapData/mapData';
-
+import { ActivatedRoute } from '@angular/router';
+import {Base64Service} from '../../_common/services/base64.service';
 import {apiPaths} from  '../../../serverConfig/apiPaths';
-import {Http, Headers} from '@angular/http';
+import {Response} from '@angular/http';
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/createProfile/component/createProfile.component.ts]: This Response not used anywhere?

File: Application/Client/src/app/_common/header/components/header.component.ts (hunk 1 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/_common/header/components/header.component.ts]: fixed

File: Application/Client/src/app/_common/header/components/header.component.ts (hunk 2 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/_common/header/components/header.component.ts]: fixed

File: Application/Client/src/app/_common/similarListing/components/similarListing.component.ts (hunk 3 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/_common/similarListing/components/similarListing.component.ts]: to check the similar listing as per user intrest

File: Application/Client/src/app/_common/similarListing/components/similarListing.component.ts (hunk 4 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/_common/similarListing/components/similarListing.component.ts]: fixed

File: Application/Client/src/app/card-list/components/card-list.component.ts (hunk 5 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/card-list/components/card-list.component.ts]: fixed

File: Application/Client/src/app/createProfile/component/createProfile.component.ts (hunk 7 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/createProfile/component/createProfile.component.ts]: Yes it is used in get method below to get the profile data

File: Application/Client/src/app/_common/header/components/header.component.ts [hunk 8]
```diff
@@ -31,11 +36,16 @@ export class HeaderComponent implements OnInit{
   }
 
     public collapsed(event:any):void {
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/_common/header/components/header.component.ts]: Remove this function as we dont need empty function

File: Application/Client/src/app/_common/header/components/header.component.ts [hunk 9]
```diff
@@ -31,11 +36,16 @@ export class HeaderComponent implements OnInit{
...
-        console.log(event);
+
     }
 
     public expanded(event:any):void {
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/_common/header/components/header.component.ts]: Same as above

File: Application/Client/src/app/card-list/components/card-list.component.ts [hunk 10]
```diff
@@ -21,17 +21,14 @@ export class CardListComponent{
     @Input() cards;
 
     private isLoading:boolean = false;
-    
-    ngOnInit() {
-        console.log('all = ',this.cards);
-    }
-        
+
+    ngOnInit() {}
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/card-list/components/card-list.component.ts]: remove this function as not needed

Comment (lakuleavadhutglobant) [reply Application/Client/src/app/card-list/components/card-list.component.ts]: fixed

File: Application/Client/src/app/_common/header/components/header.component.ts (hunk 9 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/_common/header/components/header.component.ts]: we require this function for HTML

File: Application/Client/src/app/_common/header/components/header.component.ts (hunk 8 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/_common/header/components/header.component.ts]: we require this function for header html

**Pull request merged**